examples/transmissions/send_inline_image.js

```diff
@@ -0,0 +1,43 @@
+'use strict';
+
+const key = 'YOURAPIKEY';
```

Comment:
Yeah. I purposefully changed it on this one. Since we are targeting higher node versions and `const`/`let` is preferred over `var`.